frontend/src/components/ExternalDb.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <section class="section extended-section">
+    <div v-if="externalDb" class="container is-fullhd">
+      <h3 class="title">
+        {{ externalDb.dbName }} {{ components[0].componentType }} {{ externalDb.externalId }}
+      </h3>
+      <h4 class="subtitle">
+        The following Metabolic Atlas
+        {{ components.length === 1 ? 'component is' : 'components are' }}
+        associated with this external DB reference.
+      </h4>
+      <div class="tags are-medium is-flex-direction-column is-align-items-flex-start">
+        <span v-for="c in components" :key="c.id + c.model + c.version" class="tag is-light">
+          <router-link :to="{ name: c.componentType.toLowerCase(), params: { model:
+            c.model, id: c.id } }">
+            {{ c.componentType }} {{ c.id }} from {{ c.model }} {{ c.version }}
+          </router-link>
+        </span>
```

Comment:
How about restricting the tag highlight just to the id?

```suggestion
        <span v-for="c in components" :key="c.id + c.model + c.version">
          <router-link :to="{ name: c.componentType.toLowerCase(), params: { model:
            c.model, id: c.id } }">
            {{ c.componentType }}
            <span  class="tag is-light">{{ c.id }}</span>
            from {{ c.model }} {{ c.version }}
          </router-link>
        </span>
```

Reply:
Since the rest of the link is blue text, this contrast actually makes the id look like it's the only part of the link that is not a link. I personally prefer having the whole thing as a tag, or if we only tag the id I suggest making the rest of the line not a part of the link.

Reply:
We have some tags in the GEM Browser, but they are used for the `id` only. so yeah it would make sense to make the rest of the text black.

Reply:
![image](https://user-images.githubusercontent.com/32480198/119821365-c61bb600-bef2-11eb-8fa3-32ad0748e5ea.png)
Looks good like this?

Reply:
It does to me 👍🏻

Reply:
Just realized that in the original implementation the text inside the tag give a stronger indication that they are links. Was this style intentionally removed? I mentioned a thought about spacing in [another comment](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/574#discussion_r640728837) but even if we ignore that and only have the tag on the id/name, I think it would still look better if the tags look like links. Thoughts?

<img width="369" alt="image" src="https://user-images.githubusercontent.com/423498/119959877-af359c00-bfa4-11eb-9f66-aad74bdddea6.png">

Reply:
Pushed a commit fixing so that the text inside the tag look like a link :+1: